<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowMainToolbar"/>
    <addaction name="actionShowSegmentationToolbar"/>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="title">
     <string>Segmentation</string>
    </property>
    <addaction name="actionLoadImage"/>
    <addaction name="actionSet_Number_of_Classes"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Segmentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSegmentation"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionClose"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QToolBar" name="SegmentationToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadImage"/>
   <addaction name="actionStart_Segmentation"/>
  </widget>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New Segmentation Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionShowMainToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Main Toolbar</string>
   </property>
  </action>
  <action name="actionLoadImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionShowSegmentationToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Segmentation Toolbar</string>
   </property>
  </action>
  <action name="actionSet_Number_of_Classes">
   <property name="text">
    <string>Set Number of Classes</string>
   </property>
   <property name="toolTip">
    <string>Set Number of Classes for Segmentation</string>
   </property>
  </action>
  <action name="actionSelect_Marker">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select Marker</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionStart_Segmentation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start Segmentation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
